src/manifest.js

```diff
@@ -92,6 +93,21 @@ export const parseManifest = ({
     manifest.targetDuration = targetDuration;
   }
 
+  const parts = getLastParts(manifest);
```

Comment:
set a default part target duration for manifests with parts that do not have one.